architect/systems/optical/spectrometers.py

```diff
@@ -303,14 +303,13 @@ def get_pointing_accuracy_constraint(
 
         return constraint_angle
 
-    def get_ground_target_error(self, orbital_altitude, skew_angle):
+    def get_ground_target_error(self, orbital_altitude, skew_angle, pointing_accuracy: unit.deg = 0):
         """Get the ground target error from the skew angle error.
 
-        Ref:https://www.notion.so/utat-ss/Create-tradebook-for-uncertainty-in-ground-target-vs-uncertainty-in-pointing-accuracy-ee938e1ee2fc40f2891f574ddfeac495
+        Ref: https://www.notion.so/utat-ss/Ground-Target-Error-vs-Pointing-Accuracy-22b3069f4a0344b08339e5004f90438b
 
         """
-
-        ground_error = np.tan(skew_angle) * orbital_altitude
+        ground_error = orbital_altitude * (np.tan(pointing_accuracy + skew_angle) - np.tan(pointing_accuracy))
```

Comment:
Rearrange parameters to match equation card:
```suggestion
        ground_error = orbital_altitude * (np.tan(skew_angle + pointing_accuracy) - np.tan(pointing_accuracy))
```